types/ramda/test/count-tests.ts

```diff
@@ -0,0 +1,8 @@
+import * as R from 'ramda';
+
+() => {
+  const numbers = [1.0, 1.1, 1.2, 2.0, 3.0, 2.2];
+  const letters = R.split('', 'abcABCaaaBBc');
+  R.count(R.gt(2))(numbers); // => 2
+  R.count(R.equals('a'))(letters); // => 4
+};
```

Comment:
It wold be great to add negative scenario test case with $ExpectError to cover generic